llvm/utils/TableGen/AsmMatcherEmitter.cpp

```diff
@@ -661,6 +673,11 @@ struct MatchableInfo {
     if (AsmVariantID != RHS.AsmVariantID)
       return false;
 
+    // Sort by the resultant instuctions size, eg. for ARM instructions
```

Comment:
The comment needs to be fixed. This function does not sort things.

Reply:
Done

Reply:
Done, thank you for catching.